app/src/main/java/com/doctor/yumyum/presentation/ui/mypage/MyPageViewModel.kt

```diff
@@ -52,11 +52,13 @@ class MyPageViewModel : BaseViewModel() {
 
     private fun isGradeUp(newGrade: String) {
         if (localGrade != null) {
-            if (localGrade != newGrade) {
-                _gradeUp.postValue(newGrade)
-                userRepository.setLocalGrade(newGrade)
+            if (localGrade.isNotBlank()) {
+                if (localGrade != newGrade) {
+                    _gradeUp.value = newGrade
+                }
```

Comment:
54번째 줄이랑 합쳐서 이렇게 하는 거 어떤가용
```suggestion
            if (!localGrade.isNullOrBlank() && localGrade != newGrade) {
                _gradeUp.value = newGrade
```

Reply:
수정하겠습니당!